teams/wg-mail.toml

```diff
@@ -0,0 +1,28 @@
+name = "wg-mail"
```

Comment:
Might `wg-email` (and "Email working group") be a more descriptive and clearer name?